.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-18.04, macos-10.15, windows-2019]
-        python: [3.6, 3.7, 3.8]
+        python: [3.6, 3.7, 3.8, 3.9]
```

Comment:
add comment here to remind others to change line 78 whenever they modify this

Reply:
ideally, we would want to just take 1st and last elements in the array on line 78 if that is supported